docs/README.md

```diff
@@ -70,41 +77,99 @@ root = "./app"
 
 ### Third-party modules
 
-Third-party modules are external Python packages that are not part of the standard library or your own project’s code. These are typically installed using a package manager like uv or pip and include libraries such as `requests`, `numpy`, or `django`.
+Third-party modules are Python packages that are not part of your project or the standard library.
+These are usually declared as dependencies in a `pyproject.toml` or `requirements.txt` file
+and installed using a package manager like uv or pip. Examples of popular third-party
+modules are `requests`, `numpy` and `django`.
+
+ty searches for third-party modules in the configured [Python environment](#python-environment).
+
+### Python environment
+
+The Python environment is used for discovery of third-party modules.
+
+By default, ty will attempt to discover a virtual environment.
+
+First, ty checks for an active virtual environment using the `VIRTUAL_ENV` environment variable. If
+not set, ty will search for a `.venv` directory in the project root or working directory. ty only
+supports discovery of virtual environments at this time.
 
-ty searches third-party modules in your project's virtual environment. By default, it looks for a virtual environment in a `.venv` folder located at the project root. If the `VIRTUAL_ENV` environment variable is set, ty will use the path specified there instead. If your project uses a different location for your virtual environment, specify the location by setting the [`environment.python`](./reference/configuration.md#python) configuration or [`--python`](./reference/cli.md#ty-check--python) CLI option.
+The Python environment may be explicitly configured using the
+[`environment.python`](./reference/configuration.md#python) setting or
+[`--python`](./reference/cli.md#ty-check--python) flag.
```

Comment:
```suggestion
The Python environment may be explicitly configured using the
[`environment.python`](./reference/configuration.md#python) setting or
[`--python`](./reference/cli.md#ty-check--python) flag. This takes
precedence over the virtual environment discovery detailed above.
```

Reply:
I don't want to add that :p explicit configuration _always_ takes place over defaults. That's the case for all of our options.

Reply:
eh, fair enough